src/main/java/org/jabref/gui/fieldeditors/PersonsEditor.java

```diff
@@ -28,7 +30,8 @@ public PersonsEditor(final Field field,
                 ? new EditorTextField()
                 : new EditorTextArea();
 
-        textInput.textProperty().bindBidirectional(viewModel.textProperty());
+        decoratedStringProperty = new UiThreadStringProperty(viewModel.textProperty());
```

Comment:
Is it necessary to keep a track of the reference? If not I would prefer if `decoratedStringProperty` is converted to a local variable (or be inlined completely).

Reply:
In fact, i tried it with an inlined approach first, but the list of files did not update itself then.
I did not investigate it further, but i think it has to do with `WeakReference`s used inside `Bindings.bindContentBidirectional`.